lib/rules/valid-define-options.js

```diff
@@ -12,9 +12,7 @@ module.exports = {
     type: 'problem',
     docs: {
       description: 'enforce valid `defineOptions` compiler macro',
-      // TODO Switch in the next major version
-      // categories: ['vue3-essential', 'vue2-essential'],
-      categories: undefined,
+      categories: ['vue3-essential', 'vue2-essential'],
```

Comment:
Is `defineOptions` even supported in Vue 2? If not, then it makes no sense to add this rule to the `vue2-essentials` config.

Reply:
It can only be used in `<script setup>` syntax. Removed ✅

Reply:
Oh, I'm not very familiar with Vue 2.7, I'll check whether `defineOptions` was introduced in Vue 2.7 like `defineProps`.
`vue/valid-define-emits` and `vue/valid-define-props` are in the `essential` category for both Vue 2 and Vue 3.